ai-baseline/app/main.py

```
@@ -0,0 +1,570 @@
+            if original_columns:
+                student_id_col = original_columns[0]
+                logger.info(f"Using '{student_id_col}' as student identifier column")
+        
+        answer_col = None
+        for col_name in ['student answer', 'Student Answer', 'student_answer', 'answer']:
+            if col_name in original_columns:
+                answer_col = col_name
+                break
+        
+        if not answer_col:
+            raise ValueError(f"Could not find student answer column. Available columns: {original_columns}")
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)